dev/tests/verification/TestModule/ActionGroup/BasicActionGroup.xml

```diff
@@ -13,9 +13,12 @@
 
     <actionGroup name="actionGroupWithDefaultArgumentAndStringSelectorParam">
         <arguments>
+            <!--  Comments in arguments are not affecting test generation -->
```

Comment:
Please add a **new** verification test specific to comment generation instead of reusing an old actionGroup/testcase. We should keep testcases very focused in testing responsibility so we can use them as quasi unit tests.